spark8t/services.py

```diff
@@ -1131,7 +1142,11 @@ def create(self, service_account: ServiceAccount) -> str:
             KubernetesResourceType.ROLEBINDING,
             rolebindingname,
             namespace=service_account.namespace,
-            **{"role": rolename, "serviceaccount": service_account.id},
+            **{
+                "role": rolename,
+                "serviceaccount": service_account.id,
+                "username": service_account.name,
+            },
```

Comment:
**todo:** I'm not sure I see the value of the star expansion here over just regular arguments, i.e:

```python
self.kube_interface.create(
    KubernetesResourceType.ROLEBINDING,
    rolebindingname,
    namespace=service_account.namespace,
    role=rolename,
    serviceaccount=service_account.id,
    username=service_account.name
)

Reply:
Done